action.yml

```
@@ -0,0 +1,28 @@
+name: "Connect SSM"
+description: "Github Action for connect to a remote host through aws ssm session."
+
+inputs:
+  target-instance-id:
+    description: "EC2 instance id of your SSM managed instance."
+    required: true
+  aws-region:
+    description: "AWS Region which instance is located."
+    required: true
+  aws-profile:
+    description: "AWS Profile. If not provided, the default user will be used."
+    required: true
```

```suggestion
    required: false
```

sobre isso aqui, você havia dito que tinha deixado esse input "opcional". Esse required "true" é necessário para preencher o valor como vazio  ?, se for isso mesmo, não precisa aceitar essa suggestion.